service.go

```diff
@@ -194,6 +194,37 @@ func NewSeleniumService(jarPath string, port int, opts ...ServiceOption) (*Servi
 	return s, nil
 }
 
+func NewSeleniumServiceV4(jarPath string, port int, opts ...ServiceOption) (*Service, error) {
```

Comment:
selenium_test.go needs to be updated to test Selenium v4:

https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/selenium_test.go#L22
https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/selenium_test.go#L131
https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/selenium_test.go#L161
https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/selenium_test.go#L196
https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/selenium_test.go#L227
https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/selenium_test.go#L284

We should also add Selenium v4 to be automatically downloaded by vendor/init.go:

https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/vendor/init.go#L63